packages/zoe/src/cleanProposal.js

```diff
@@ -52,18 +60,16 @@ const cleanKeys = (allowedKeys, record) => {
 export const getKeywords = keywordRecord =>
   harden(Object.getOwnPropertyNames(keywordRecord));
 
-export const coerceAmountKeywordRecord = (
-  getAmountMath,
-  allegedAmountKeywordRecord,
-) => {
+export const coerceAmountKeywordRecord = allegedAmountKeywordRecord => {
```

Comment:
No one imports this. If there's no reason to export it, I'd feel more comfortable if we did not.
```suggestion
const coerceAmountKeywordRecord = allegedAmountKeywordRecord => {
```